app/controllers/puppies_controller.rb

```diff
@@ -3,13 +3,15 @@ class PuppiesController < ApplicationController
   skip_before_action :authenticate_user!, only: [:show, :index]
 
   def index
+
```

Comment:
I just realized (and I do definitely include me in this as well), that we never write any comments in our code. I guess for next week we should start doing that more often!